stdlib/SparseArrays/src/higherorderfns.jl

```diff
@@ -1006,10 +1007,6 @@ end
     _copyto!(parevalf, dest, passedsrcargstup...)
 end
 
-struct CapturedScalars{F, Args, Order}
```

Comment:
`git grep CapturedScalars` shows me that there is no code using it.  Maybe some remnant from experimenting with `capturescalars`?  This was irrelevant to this PR but I thought it's OK to remove it here (as I'm touching `capturescalars`).